apps/epic-react/.env.production

```diff
@@ -0,0 +1,33 @@
+## ⚠️ PRODUCTION ⚠️ ##
+
+# General Metadata
+NEXT_PUBLIC_APP_NAME=epic-react
+NEXT_PUBLIC_SITE_TITLE="Epic React"
+NEXT_PUBLIC_HOST=epicreact.dev
+NEXTAUTH_URL=https://epicreact.dev
+NEXT_PUBLIC_URL=https://epicreact.dev
+NEXT_PUBLIC_SUPPORT_EMAIL=team@epicreact.dev
+NEXT_PUBLIC_PARTNER_FIRST_NAME=Kent
+NEXT_PUBLIC_PARTNER_LAST_NAME=C. Dodds
+NEXT_PUBLIC_PARTNER_TWITTER=kentcdodds
+NEXT_PUBLIC_SEO_KEYWORDS="react, javascript, web development"
+NEXT_PUBLIC_PRODUCT_DESCRIPTION="Get Really Good at React"
+
+# Google Analytics
+NEXT_PUBLIC_GOOGLE_ANALYTICS=CHANGE_ME
+
+# Transactional Email
+EMAIL_SERVER_HOST=smtp.postmarkapp.com
+EMAIL_SERVER_PORT=587
+
+# ConvertKit Newsletter Form
+NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM=FORM_ID
+NEXT_PUBLIC_CONVERTKIT_TOKEN=CK_TOKEN
+NEXT_PUBLIC_CONVERTKIT_ANSWER_URL=/api/skill/answer/convertkit
+NEXT_PUBLIC_CONVERTKIT_SUBSCRIBE_URL=/api/skill/subscribe/convertkit
+NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL=/api/skill/subscriber/convertkit
+
+# Sanity
+NEXT_PUBLIC_SANITY_API_VERSION=2023-01-03
+NEXT_PUBLIC_SANITY_PROJECT_ID=project_id_from_sanity
```

Comment:
replace this and `.env.development` with the project ID for the Epic React sanity project once that is setup